test/Api.Test/Api.Test.csproj

```diff
@@ -19,6 +19,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Api\Api.csproj" />
     <ProjectReference Include="..\..\src\Core\Core.csproj" />
+    <ProjectReference Include="..\Core.Test\Core.Test.csproj" />

```

Comment:
It doesn't look like these tests depend on this project.

I think you probably stuck this in here wanting to use some of the AutoFixture stuff, but didn't end up doing it.

If we end up needing that code, we should make a new project to store cross-project testing code and customizations rather than reference other test projects.

Reply:
Looks like I kept this in to resolve the Autofixture.Xunit2 references. I've now swapped it out for the Xunit2 NuGet package, which is the right way to include it.